Dockerfile.simple

```diff
@@ -19,6 +19,7 @@ RUN sed -ri "s/(httpredir|deb).debian.org/$APT_MIRROR/g" /etc/apt/sources.list
 # https://github.com/docker/docker/blob/master/project/PACKAGERS.md#runtime-dependencies
 RUN apt-get update && apt-get install -y --no-install-recommends \
 		btrfs-tools \
```

Comment:
This is the package that used to contain this header, so we should be able to remove it from the list now (also, this list is in sorted order). 👍

Reply:
@tianon  Thanks, updated